src/TelescopeExtensionServiceProvider.php

```diff
@@ -28,9 +30,14 @@ public function boot(): void
             __DIR__ . '/../config/telescope-guzzle-watcher.php' => config_path('telescope-guzzle-watcher.php'),
```

Comment:
could we also move this config publishing to the first publishing on the line 23?